core/domain/suggestion_services.py

```diff
@@ -2904,3 +2908,333 @@ def enqueue_contributor_ranking_notification_email_task(
+        <div class="signature" style="width: 50%; float:left; 
+        text-align: center;"><div style="
+        font-family: Brush Script MT, Brush Script Std, cursiv;
+        font-size: 36px; line-height: 10px; margin-top: 0px;
+        color: #000000;">"""+ signature +"""</div> <center> 
+        <p class="line" style="width: 186px; height: 0px; border: 1px 
+        solid #00645C;"></p> </center> <p style="font-family: 'Roboto'; 
+        font-style: normal; font-weight: 400; font-size: 24px; 
+        line-height: 28px; margin-top: 0px; color: #8F9899;">SIGNATURE</p> 
+        </div> <div class="date" style="width: 50%; float:left; 
+        text-align: center;"><div style="font-family: 'Roboto';
+        font-size: 24px; color: #000000; line-height: 10px;">
+        """ + date + """</div><center> <p class="line" style="
+        width: 186px; height: 0px; border: 1px solid #00645C;"></p> 
+        </center> <p style="font-family: 'Roboto'; font-style: normal; 
+        font-weight: 400; font-size: 24px; line-height: 28px; 
+        margin-top: 0px; color: #8F9899;">DATE</p> </div> </div> </div>
+        </div> </div> </div> </body> </html>
+    """
+
+    return translation_submitter_certificate_template
+
+
+def _generate_question_contributor_certificate(
+    from_date: datetime.datetime,
+    to_date: datetime.datetime,
+    username: str,
+    user_id: str,
+    date: str
+) -> str:
+    """Generates a certificate for a user's question contributions.
+
+    Args:
+        suggestion_type: str. The type of suggestion that the certificate
+            needs to generate.
+        from_date: datetime.datetime. The start of the date range for which
+            the contributions were created.
+        to_date: datetime.datetime. The end of the date range for which
+            the contributions were created.
+        username: str. The username of the contributor.
+        user_id: str. The user ID of the contributor.
+        date: datetime.datetime. The current date that should be included in the
+            certificate.
+
+    Returns:
+        str. The HTML template to generate the certificate.
+
+    Raises:
+        Exception: The suggestion type given to generate the certificate is
+            invalid.
+    """
+    signature = feconf.QUESTION_TEAM_LEAD
+
+    suggestions = (
+        suggestion_models.GeneralSuggestionModel
+            .get_question_suggestions_submitted_within_given_dates(
+                from_date, to_date, user_id))
+
+    minutes_contributed = 0
+    for model in suggestions:
+        suggestion = get_suggestion_from_model(model)
+
+        content = suggestion.change.question_dict[
+            'question_state_data']['content']['html']
+
+        if 'oppia-noninteractive-image' in content:
+            minutes_contributed += 20
+        else:
+            minutes_contributed += 12
+    # Source: https://github.com/oppia/oppia/pull/16513/files#r1027109026
```

Comment:
ditto as above

Reply:
Ditto as above the link is not working for me. Also should this comment be put above line 3149?